### PR TITLE
added commands, exception, result parceable

### PR DESCRIPTION
**Pull request by anti43**
sorry for this huge PR again, really I have even more and have to cherry pick already

### CODE REVIEW

File: src/main/java/pt/lighthouselabs/obd/commands/control/AvailablePidsObdCommand.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+/**
+ * Fuel systems that use conventional oxygen sensor display the commanded open
+ * loop equivalence ratio while the system is in open loop. Should report 100%
+ * when in closed loop fuel.
+ * <p>
+ * To obtain the actual air/fuel ratio being commanded, multiply the
+ * stoichiometric A/F ratio by the equivalence ratio. For example, gasoline,
+ * stoichiometric is 14.64:1 ratio. If the fuel control system was commanded an
+ * equivalence ratio of 0.95, the commanded A/F ratio to the engine would be
+ * 14.64 * 0.95 = 13.9 A/F.
```
Comment (nomwerp) [inline src/main/java/pt/lighthouselabs/obd/commands/control/AvailablePidsObdCommand.java]: comment doesn't apply to this command?

Comment (anti43) [inline src/main/java/pt/lighthouselabs/obd/commands/control/AvailablePidsObdCommand.java]: I just realized that all the commands are not updated

File: src/main/java/pt/lighthouselabs/obd/commands/engine/AbsoluteLoadObdCommand.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+   * @return a double.
+   */
+  public double getRatio() {
+    return percentage;
+  }
+
+  @Override
+  public String getName() {
+    return AvailableCommandNames.EQUIV_RATIO.getValue();
+  }
```
Comment (nomwerp) [inline src/main/java/pt/lighthouselabs/obd/commands/engine/AbsoluteLoadObdCommand.java]: is that supposed to be EQUIV_RATIO here?

File: src/main/java/pt/lighthouselabs/obd/commands/fuel/FuelTrimObdCommand.java [hunk 3]
```diff
@@ -46,27 +50,27 @@ private float prepareTempValue(final int value) {
...
   }
 
-  @Override
-  public String getFormattedResult() {
-    return String.format("%.2f%s", fuelTrimValue, "%");
-  }
+    @Override
+    public String getFormattedResult() {
+        return String.format("%.2f%s", percentage, "%");
+    }
```
Comment (nomwerp) [inline src/main/java/pt/lighthouselabs/obd/commands/fuel/FuelTrimObdCommand.java]: seems to mix of 4 and 2 space indents , would you mind reformatting this bit to two?

**Comment by nomwerp** [comment]
thanks, quickly reviewed it and looks good to me , have not tried to build it.

**Comment by anti43** [comment]
@nomwerp i think i merge-broke soemthing, let me just verify all the commands again

**Comment by pires** [comment]
@nomwerp please merge only after building and running tests.

**Pull request closed without merging**